app/views/spree/admin/products/_group_buy_form.html.haml

```diff
@@ -1,14 +1,14 @@
 = f.field_container :group_buy do
-  = f.label :group_buy, 'Group buy?'
+  = f.label :group_buy, t('.group_buy')
   %br
   .alpha.two.columns
-    = f.radio_button :group_buy, '1', :checked => f.object.group_buy
-    = f.label :group_buy_1, 'Yes'
+    = f.radio_button :group_buy, '1', checked: f.object.group_buy
+    = f.label :group_buy_1, t('yes')
   .alpha.two.columns
-    = f.radio_button :group_buy, '0', :checked => !f.object.group_buy
-    = f.label :group_buy_0, 'No'
+    = f.radio_button :group_buy, '0', checked: !f.object.group_buy
+    = f.label :group_buy_0, t('no')
```

Comment:
As a suggestion, I'd rather use symbols for these two keys.

Reply:
Hi @sauloperez I've just fixed and changed it to symbols. Tks. Leandro 👍 😄